pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -838,26 +838,27 @@ func (r *ReconcileClusterDeployment) reconcileInstallingClusterInstall(cd *hivev
 	return r.reconcileExistingInstallingClusterInstall(cd, logger)
 }
 
-func dnsZoneNotReadyMaybeTimedOut(cd *hivev1.ClusterDeployment, logger log.FieldLogger) (string, string, time.Duration) {
+func dnsZoneNotReadyMaybeTimedOut(cd *hivev1.ClusterDeployment, logger log.FieldLogger) (string, string, time.Duration, bool) {
 	cond := controllerutils.FindCondition(cd.Status.Conditions, hivev1.DNSNotReadyCondition)
+	var provisionStoppedChanged bool
```

Comment:
We could really do without this variable outside of the `timeToTimout` condition; just inline `false` everywhere except L861/878. Stylistic nit, feel free to ignore.